Purchases/Public/RCPurchases.m

```diff
@@ -593,14 +602,18 @@ - (void)restoreTransactionsWithCompletionBlock:(nullable RCReceivePurchaserInfoB
                             discounts:nil
           presentedOfferingIdentifier:nil
                          observerMode:!self.finishTransactions
-                 subscriberAttributes:nil
+                 subscriberAttributes:subscriberAttributes
                            completion:^(RCPurchaserInfo *_Nullable info, NSError *_Nullable error) {
                                [self dispatch:^{
                                    if (error) {
+                                       if (error.successfullySynced) {
+                                           [self markAttributesAsSynced:subscriberAttributes appUserID:self.appUserID];
+                                       }
```

Comment:
@aboedo Sorry to comment here after merging, but this is not printing any subscriber attribute errors that the backend might send down. Do you think we should?

Reply:
great catch. I'll update in a new PR